apps/chat/migrations/0010_celery_periodic.py

```diff
@@ -0,0 +1,34 @@
+# Generated by Django 4.1.7 on 2023-04-05 06:47
+
+from django.db import migrations
+
+
+def create_celery_schedules(apps, schema_editor):
```

Comment:
Out of curiosity, is there a specific reason why you're doing it this way instead of the way it was?

Reply:
Cool, thanks! Another benefit is the ability to control this via the admin page if we wanted to.